### PR TITLE
added timer and messaging box

### PR DESCRIPTION
**Pull request by HARSHV10**
## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: src/components/Lounge.js [hunk 1]
```diff
@@ -1,12 +1,86 @@
...
+    dangerouslySetInnerHTML={{
+      __html:
+        "\n\t\t\th1 {\n\t\t\t\tcolor:white;\n\t\t\t}\n\t\t\tdiv.gfg {\n\t\t\t\tmargin:5px;\n\t\t\t\tpadding:5px;\n\t\t\t\tcolor: white;\n\t\t\t\twidth: 250px;\n\t\t\t\theight: 100%;\n\t\t\t\toverflow: auto;\n\t\t\t\ttext-align:justify;\n        position:fixed;\n        \n\n\t\t\t}\n\t\t"
+    }}
+  />
+    <div className="gfg scroll">
+        <h1 className='msg'>MESSAGE</h1>
+
+
+        <hr></hr><hr></hr>
```
Comment (Zoronium) [inline src/components/Lounge.js]: Please State the use of these hr tags and remove them if they are redundant

File: src/components/Lounge.js [hunk 2]
```diff
@@ -1,12 +1,86 @@
...
+ 
+        <h3 className='messegeBox'><p className='msg'>messeng</p></h3>
+        <h3 className='messegeBox'><p className='msg'>messeng</p></h3>
+        <h3 className='messegeBox'><p className='msg'>messeng</p></h3>
```
Comment (Zoronium) [inline src/components/Lounge.js]: Why do so many message boxes, remove any more than required?

**Review by Zoronium — CHANGES_REQUESTED** [review]
Please check these changes and add any screenshots.

File: src/components/Lounge.js (hunk 1 above)
Comment (HARSHV10) [reply src/components/Lounge.js]: these <hr> tag are the horizontal rules under message text for decoration purposes

File: src/components/Lounge.js (hunk 2 above)
Comment (HARSHV10) [reply src/components/Lounge.js]: these are testing messages for the messaging box thus all would be replaced by the actual messages

**Review by Zoronium — COMMENTED** [review]
Looks good 👍🏻

**Pull request merged**